pandas/core/indexes/base.py

```diff
@@ -2316,16 +2316,29 @@ def shift(self, periods=1, freq=None):
 
     def argsort(self, *args, **kwargs):
         """
-        Returns the indices that would sort the index and its
-        underlying data.
+        Return the order of the indices that would sort the index.
+
+        Parameters
+        ----------
+        *args
```

Comment:
@jorisvandenbossche this is the correct format for args/kwargs?

Reply:
Yes, I was only thinking it might actually make more sense to list the ones that are useful. I think it is only `kind` ? (`axis` certainly not, and `order` not as well?)

Reply:
yes we do this for Series.argsort now. so should replicate here. or shared these doc-strings (best)